components/status/StatusBar.vue

```diff
@@ -1,19 +1,46 @@
 <template>
   <div class="div__status-bar">
     <span class="span__status-bar-text">{{ alert_txt }}</span>
+    <span>
+      <b-modal
+        id="error-catch"
+        size="sm"
+        hide-footer
+        hide-header
+        hide-header-close
+        :static="true"
+      >
+        <ErrorCatchWidget
+          id="error"
+          :log_filepath="log_path"
+          @ok-clicked="remove_errorcatch"
```

Comment:
`errorcatch` should be two words.  just like how it is two words in the name of the widget when in PascalCase: `ErrorCatch`

Reply:
Agreed.